src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutesSpec.scala

```diff
@@ -103,4 +102,62 @@ class GoogleExtensionRoutesSpec extends FlatSpec with Matchers with ScalatestRou
       }
     }
   }
+
+  "GET /api/google/policy/{resourceTypeName}/{resourceId}/{accessPolicyName}/sync" should "200 with sync date" in {
+    val resourceType = ResourceType(ResourceTypeName("rt"), Set(ResourceAction("alter_policies"), ResourceAction("can_compute"), ResourceAction("read_policies")), Set(ResourceRole(ResourceRoleName("owner"), Set(ResourceAction("alter_policies"), ResourceAction("read_policies")))), ResourceRoleName("owner"))
+    val resourceTypes = Map(resourceType.name -> resourceType)
+    val defaultUserInfo = UserInfo("accessToken", WorkbenchUserId("user1"), WorkbenchUserEmail("user1@example.com"), 0)
+    val googleDirectoryDAO = new MockGoogleDirectoryDAO()
+    val directoryDAO = new MockDirectoryDAO()
+    val googleIamDAO = new MockGoogleIamDAO()
+    val policyDAO = new MockAccessPolicyDAO()
+    val pubSubDAO = new MockGooglePubSubDAO()
+    val googleExt = new GoogleExtensions(directoryDAO, policyDAO, googleDirectoryDAO, pubSubDAO, googleIamDAO, googleServicesConfig, petServiceAccountConfig)
+    googleExt.onBoot()
+    val mockResourceService = new ResourceService(resourceTypes, policyDAO, directoryDAO, googleExt, "example.com")
+    val samRoutes = new TestSamRoutes(mockResourceService, new UserService(directoryDAO, googleExt, googleDirectoryDAO, "dev.test.firecloud.org"), new StatusService(directoryDAO, googleDirectoryDAO), UserInfo("", defaultUserInfo.userId, defaultUserInfo.userEmail, 0)) with GoogleExtensionRoutes {
+      val googleExtensions = googleExt
+    }
+
+    //create user
+    Post("/register/user") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.Created
+    }
+
+    Get(s"/api/google/resource/${resourceType.name}/foo/${resourceType.ownerRoleName.value}/sync") ~> samRoutes.route ~> check {
+/* hasn't been created yet so should return not found but doesn't
```

Comment:
Would like to fix this. Also this could be a separate test.

Reply:
I think it is good to have in this test as it establishes the baseline

Reply:
We can't actually do this check in this test the way it's structured. The sync status is known by DirectoryDAO while existence of the resource is known by some other DAO. Since the mocks aren't linked (and shouldn't be or else they start to be a real alternate implementation) MockDirectoryDAO can't reply one way for a non-existent group and another way for a created-but-not-synced group.